obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  status: 1
+- agasc_id:
+  - 276562048
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276695120
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276695632
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276700592
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276701784
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276702368
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 276703712
+  comments: Normal Sun Mode
+  mp_starcat_time: 2015:006:07:59:28.866
+  status: 1
+- agasc_id:
+  - 788418168
+  comments: Never acquired
+  mp_starcat_time: 2015:014:04:32:44.554
+  status: 1
+- agasc_id:
+  - 788418168
+  comments: Never acquired
+  mp_starcat_time: 2015:022:03:00:16.542
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:024:19:18:39.261
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:025:03:12:57.957
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:025:05:06:07.571
+  status: 1
+- agasc_id:
+  - 111807640
+  comments: Faint star. Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:032:15:55:37.026
+  status: 1
+- agasc_id:
+  - 288753656
+  comments: Faint star. Tracking affected by imposter. Set magnitude to MAXMAG
+  mp_starcat_time: 2015:125:13:11:24.129
+  status: 1
+- agasc_id:
+  - 881460168
+  comments: Faint star. Affected by imposters
+  mp_starcat_time: 2015:130:19:50:00.443
+  status: 1
+- agasc_id:
+  - 1092505568
+  comments: Faint star. Affected by imposters
+  mp_starcat_time: 2015:159:15:26:54.646
+  status: 1
+- agasc_id:
+  - 262408104
+  comments: Short run. Magnitude seems ok
+  mp_starcat_time: 2015:201:22:05:03.421
+  status: 0
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:212:20:00:08.125
+  status: 1
+- agasc_id:
+  - 821318496
+  comments: Faint star. Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2015:220:11:07:04.609
+  status: 1
+- agasc_id:
+  - 500706352
+  comments: Failed to track but there are snapshots
+  mp_starcat_time: 2015:231:00:04:31.256
+  status: 0
+- agasc_id:
+  - 144714600
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:233:22:11:26.142
+  status: 1
+- agasc_id:
+  - 144714600
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:234:03:22:43.521
+  status: 1
+- agasc_id:
+  - 376724944
+  comments: Mag 6.5 spoiler 45 arcsec away
+  mp_starcat_time: 2015:331:14:10:26.457
+  status: 1
+- agasc_id:
+  - 195167088
```

Comment:
It looks like some of the other observations are reasonable.  Maybe this doesn't need to be set to maxmag?

Reply:
Doesn't really matter in this case.